application/modules/home/controllers/Home.php

```diff
@@ -43,4 +43,19 @@ public function index() {
 		// load view
 		$this->load_view('top', $this->data);
 	}
-}
\ No newline at end of file
+
+	/**
+	 * Test side ci
+	 *
+	 * @return void
+	 */
+	public function side_ci()
+	{
+		$this->data['exhibition']['name'] = 'test';
+
+		// load view
+		$this->load_view('side_ci', $this->data);
+	}
```

Comment:
__[code_sniffer]__
* Closing brace of a class must be followed by a single blank line; found 0
* Expected //end class

<!-- Comment by [SideCI](https://sideci.com) -->